examples/volumes/vsphere/README.md

```diff
@@ -94,6 +94,11 @@
           volumePath: "[datastore1] volumes/myDisk"
           fsType: ext4
       ```
+      In the above example datastore1 not placed within any Folder or Datastore Cluster. If datastore is placed within Datastore Cluster or some folder, appropriate path should be provided in the VolumePath as below. 
```

Comment:
clarified note with mentioning root folder - the datacenter folder

> 
In the above example datastore1 is located in the root folder. If datastore is member of Datastore Cluster or located in sub folder, the folder path needs to be provided in the VolumePath as below